wit/handlers.wit

```diff
@@ -0,0 +1,49 @@
+use wasi:http/types@0.2.0 as http-types;
+
+interface http-handlers {
+    use http-types.{incoming-request, incoming-response};
+    use types.{handler-output, decision, outcome, param, verdict};
+
+    /// Called once when the plugin is first instantiated.
+    handle-init: func() -> result<_, error>;
+    /// Called on every request.
+    ///
+    /// Plugins should use this handler to perform request enrichment
+    /// by adding new parameters to the handler output. Plugins should not
```

Comment:
Agreed, though I suspect examples will be better located in the SDK docs rather than the WIT files.

The good news is that even if someone did make the mistake, it'd probably still be fine 99% of the time, but that also likely makes it really difficult to debug if it does happen.